extension/services/spotify/player.js

```diff
@@ -53,6 +52,10 @@ this.player = (function() {
       const button = this.querySelector(selector);
       button.click();
     }
+
+    action_playAlbum({ query, thenPlay }) {
+      this.action_search({ query, thenPlay });
```

Comment:
These should be acting differently than a normal search. In the case of Spotify if you search for something, then do "See all" under albums, you'll get to this URL: https://open.spotify.com/search/dark%20side%20of%20the%20moon/albums

Or this for playlists: https://open.spotify.com/search/dark%20side%20of%20the%20moon/playlists

Similarly here's something for SoundCloud: https://soundcloud.com/search/albums?q=please%20excuse%20me%20for%20being%20antisocial

Right now we're doing everything in-page when searching, but it might be better to simply cause a page load and navigate directly to these URLs. Note that content scripts will be lost as a result and will have to be re-added.

Reply:
@ianb https://open.spotify.com/search/dark%20side%20of%20the%20moon/albums, https://open.spotify.com/search/dark%20side%20of%20the%20moon/albums shows a 404 error while accessing it.

Reply:
You're right! I guess Spotify made direct navigation to the general search work, but not these specific searches. Someone should report the bug to them ;)

It's a bummer we can't trigger an internal URL navigation, but it looks like this will require clicking on "Albums"

Reply:
Yes. I will do that.

Reply:
added clicking in the card to get the songs list.

Reply:
@ianb Added clicking on the card to get the list of songs.